services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/student_overview_table.jsx

```diff
@@ -53,6 +53,38 @@ export default class StudentOveriewTable extends React.Component {
     }
   }
 
+  activityClassificationIcon(id) {
```

Comment:
This feels like maybe it should be a Shared function, but this seems reasonable for now given that we're only using it in this one place.